library/swift/EngineBuilder.swift

```diff
@@ -339,6 +352,7 @@ open class EngineBuilder: NSObject {
       dnsFailureRefreshSecondsMax: self.dnsFailureRefreshSecondsMax,
       dnsQueryTimeoutSeconds: self.dnsQueryTimeoutSeconds,
       dnsPreresolveHostnames: self.dnsPreresolveHostnames,
+      enableInterfaceBinding: self.enableInterfaceBinding,
```

Comment:
for consistency with the existing `adminInterfaceEnabled` we should consider calling it `interfaceBindingEnabled`. Alternatively, we can change the name of `adminInterfaceEnabled` in future PRs.

Reply:
I guess I have a slight preference for the active voice, and a greater preference for addressing it later. :)